homeassistant/config_entries.py

```diff
@@ -1030,6 +1040,14 @@ async def async_finish_flow(
             ):
                 self.async_abort(progress_flow["flow_id"])
 
+            # Abort any flows in progress for the same handler when integration allows only one instance
+            if progress_flow[
+                "flow_id"
```

Comment:
Maybe use a walrus for this dict lookup since we use it both here, above and below.